cacheops/invalidation.py

```diff
@@ -40,6 +41,7 @@ def invalidate_dict(model, obj_dict):
     model = non_proxy(model)
     renamed_keys = load_script('invalidate')(args=[
         model._meta.db_table,
+        '%08x' % random.randrange(16**8),
```

Comment:
@nicwolff , hey, let's just use 4294967296 to avoid doing power calc. on every call in addition to random?

Reply:
It's a lot clearer this way since the `8` is obvs the char length, and makes no measurable time difference since '**8' is just '<<3' and the `randrange` and hex formatting take so much more time. It'd save something like .00000003 seconds per call.

Reply:
Okay